kotlin/services/ecr/src/main/kotlin/com/example/ecr/HelloECR.kt

```diff
@@ -0,0 +1,42 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.example.ecr
+
+// snippet-start:[ecr.kotlin_hello.main]
+import aws.sdk.kotlin.services.ecr.EcrClient
+import aws.sdk.kotlin.services.ecr.model.ListImagesRequest
+import kotlin.system.exitProcess
+
+suspend fun main(args: Array<String>) {
+    val usage = """
+            Usage: <repositoryName> 
+
+            Where:
+               repositoryName - The name of the Amazon ECR repository. 
+            
+    """.trimIndent()
+
+    if (args.size != 1) {
+        println(usage)
+        exitProcess(1)
+    }
+
+    val repoName = args[0]
+    listImageTags(repoName)
+}
+
+suspend fun listImageTags(repoName: String?) {
+    val listImages =
```

Comment:
I think the naming here could be a little bit better so it won't be confused with ecrClient.listImages